### PR TITLE
Add initial db setup with initial model

### PR DESCRIPTION
**Pull request by RubenSousaDinis**

### CODE REVIEW

File: frontend/prisma/schema.prisma [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+model User {
+  createdAt  DateTime @default(now())
+  updatedAt  DateTime @updatedAt
+  wallet     String   @id @db.VarChar(50)
+  questioner Question[] @relation("questioner")
+  replier    Question[] @relation("replier")
+}
+
+model Question {
```
Comment (verdefred) [inline frontend/prisma/schema.prisma:21]: missing `updatedAt`?

File: frontend/app/[wallet]/components/chat-tab.tsx [hunk 2]
```diff
@@ -1,56 +1,89 @@
...
-    if (!wallet) return ("Buidler");
-    if (!ensName) return (wallet.slice(0, 12) + "..." + wallet.slice(-10));
+    if (!wallet) return "Buidler";
+    if (!ensName) return wallet.slice(0, 12) + "..." + wallet.slice(-10);
     return ensName;
-  }
+  };
+
+  const sendQuestion = async (questionContent: string) => {
+    const response = await axios.post("/api/questions", {
```
Comment (RubenSousaDinis) [inline frontend/app/[wallet]/components/chat-tab.tsx:53]: Example: create question endpoint

File: frontend/app/[wallet]/components/chat-tab.tsx [hunk 3]
```diff
@@ -1,56 +1,89 @@
...
+      questionContent,
+      replierWallet: wallet,
+      questionerWallet: address
+    });
+
+    console.log(response.data);
+  };
+
+  const sendReply = async (questionId: number, questionAnswer: string) => {
+    const response = await axios.put(`/api/questions/${questionId}`, {
```
Comment (RubenSousaDinis) [inline frontend/app/[wallet]/components/chat-tab.tsx:63]: Example: update question endpoint

File: frontend/app/[wallet]/components/chat-tab.tsx [hunk 4]
```diff
@@ -1,56 +1,89 @@
...
+  }, [wallet]);
+
+  useEffect(() => {
+    if (wallet && address) {
+      getQuestions(wallet, address);
+    }
+  }, [wallet, address]);
+
+  const getQuestions = async (wallet: string, address: string | undefined) => {
+    const response = await axios.get(`/api/questions?questionerWallet=${address}&replierWallet=${wallet}`);
```
Comment (RubenSousaDinis) [inline frontend/app/[wallet]/components/chat-tab.tsx:41]: Example: get questions endpoint

File: frontend/app/page.tsx [hunk 5]
```diff
@@ -50,31 +54,47 @@ export default function Home() {
...
+
+  useEffect(() => {
+    if (!address) return;
+    if (user) return;
+
+    upsertUser(address);
+  }, [address]);
+
+  const upsertUser = async (address: string | undefined) => {
+    const response = await axios.put("/api/users", {
```
Comment (RubenSousaDinis) [inline frontend/app/page.tsx:84]: Example: create user endpoint

**Pull request merged**